src/MDFSpritedAnimationView.m

```diff
@@ -80,7 +80,7 @@ - (instancetype)initWithSpriteSheetImage:(UIImage *)spriteSheetImage
 - (CGSize)intrinsicContentSize {
   if (_spriteSheetImage) {
     CGFloat width = _spriteSheetImage.size.width;
-    return CGSizeMake(width, width);
+    return CGSizeMake(width, _spriteSheetImage.size.height / _numberOfFrames);
```

Comment:
can you guard against divide by zero please?

Reply:
Done, but there are other divisions happening across this files that are not guarded, so I think it's implicitly assumed that numberOfFrames is always greater or equal to 1.